ansible/roles/cphc-migration/templates/cphc_connection_check.sh

```diff
@@ -1,19 +1,42 @@
 #! /usr/bin/bash
+# alerts configuration
+MIN_FAILURES_FOR_ALERT=2
```

Comment:
We could prefix all with `ALERT_` to avoid need for comment.

`ALERT_MIN_FAILURES`
`ALERT_FREQUENCY_SECONDS`
`ALERT_STATUS_CHECK_FREQUENCY`
`ALERT_MESSAGE`

Reply:
Thanks 🙏🏼